src/components/DateInput/index.js

```diff
@@ -73,7 +73,7 @@ function splitDateString(str) {
 function getDate(input) {
   if (!input) {
     const now = new Date()
-    return [now.getFullYear(), now.getMonth(), now.getDate()]
+    return [null, now.getMonth(), null]
```

Comment:
Why not put January as month?

Reply:
oh yeah you're right 😅that was my initial thought as well

Reply:
Fixed!